src/ReaderScreen/styles/nightMode.js

```diff
@@ -0,0 +1,8 @@
+import colors from "../../common/colors";
+
+export const nightColors = (isNightMode) => ({
+  backgroundColor: isNightMode
+    ? colors.READER_STATUS_BAR_COLOR_NIGHT_MODE
+    : colors.READER_STATUS_BAR_COLOR,
+  safeAreaViewBack: { backgroundColor: isNightMode ? colors.NIGHT_BLACK : colors.WHITE_COLOR },
```

Comment:
We are using same condition twice in object, can we use it one level up, and combine both properties together. That way it would require only one conditional statement.

Reply:
fixed